CLAUDE.md

```
@@ -317,4 +318,5 @@ This file contains high-level guidelines for Claude Code. For **more detailed, t
 
 
 Random other things you should remember:
-- App\Models\Application::team must return a relationship instance., always use team()
\ No newline at end of file
+- App\Models\Application::team must return a relationship instance., always use team()
+- Always use `Model::ownedByCurrentTeamCached()` instead of `Model::ownedByCurrentTeam()->get()` for team-scoped queries to avoid duplicate database queries
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)